src/components/chatContent.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import user from './user'
+
+
+function chatContent({chatList}) {
```

Comment:
아래 map함수에서 사용해주신 것처럼, ES6의 화살표 문법을 써도 좋을 것 같아요~
```suggestion
const ChatContent = ({chatList}) => {
```
컴포넌트 첫 글자가 다른 컴포넌트에서 import될 때는 대문자로 불러 주셨던데, ChatContent 컴포넌트에서는 소문자로 선언되어 있네요..! 해당 부분 때문에 vercel 배포에서 오류가 생겼을 수 있을 것 같아요!